src/services/agent-service-constants.ts

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_PORT: number = 5338 // BEES
```

Comment:
these had to be explicitly split out because they are imported into `PercyAgentClient`.